src/test/java/uk/gov/ons/census/notifyprocessor/service/EnrichedFulfilmentRequestServiceTest.java

```diff
@@ -51,4 +51,46 @@ public void testProcessMessageNotifyApiServiceFails() throws NotificationClientE
 
     underTest.processMessage(enrichedFulfilmentRequest);
   }
+
+  @Test
+  public void testProcessMessageNotifyApiServiceFailsBadPhoneNumber()
+      throws NotificationClientException {
+    EasyRandom easyRandom = new EasyRandom();
+    NotificationClientApi notificationClientApi = mock(NotificationClientApi.class);
+    EnrichedFulfilmentRequestService underTest =
+        new EnrichedFulfilmentRequestService(notificationClientApi, "testSenderId");
+    NotificationClientException notificationClientException =
+        new NotificationClientException(
+            "Status code: 400 {\"errors\":[{\"error\":\"ValidationError\","
+                + "\"message\":\"phone_number Not a valid country prefix\"}],\"status_code\":400}");
+
+    when(notificationClientApi.sendSms(
+            anyString(), anyString(), anyMap(), anyString(), anyString()))
+        .thenThrow(notificationClientException);
+    EnrichedFulfilmentRequest enrichedFulfilmentRequest =
+        easyRandom.nextObject(EnrichedFulfilmentRequest.class);
+
+    underTest.processMessage(enrichedFulfilmentRequest);
+  }
+
+  @Test
+  public void testProcessMessageNotifyApiServiceFailsNonUkPhoneNumber()
+      throws NotificationClientException {
+    EasyRandom easyRandom = new EasyRandom();
+    NotificationClientApi notificationClientApi = mock(NotificationClientApi.class);
+    EnrichedFulfilmentRequestService underTest =
+        new EnrichedFulfilmentRequestService(notificationClientApi, "testSenderId");
+    NotificationClientException notificationClientException =
+        new NotificationClientException(
+            "Status code: 400 {\"errors\":[{\"error\":\"BadRequestError\","
+                + "\"message\":\"Cannot send to international mobile numbers\"}],\"status_code\":400}");
+
+    when(notificationClientApi.sendSms(
+            anyString(), anyString(), anyMap(), anyString(), anyString()))
+        .thenThrow(notificationClientException);
+    EnrichedFulfilmentRequest enrichedFulfilmentRequest =
+        easyRandom.nextObject(EnrichedFulfilmentRequest.class);
+
+    underTest.processMessage(enrichedFulfilmentRequest);
```

Comment:
Not sure what these tests are actually verifying? If they're just there to make use the `processMessage` call doesn't throw an exception I think they need a comment to that effect, otherwise they don't seem to be making any assertions?